tsdb/compact.go

```diff
@@ -488,16 +488,22 @@ func (c *LeveledCompactor) Write(dest string, b BlockReader, mint, maxt int64, p
 	}
 
 	if meta.Stats.NumSamples == 0 {
-		return ulid.ULID{}, nil
+		uid = ulid.ULID{}
+		level.Info(c.logger).Log(
+			"msg", "write block resulted in empty block",
```

Comment:
```suggestion
			"msg", "compaction resulted in empty block",
```

Since it's called by LeveledCompactor, I think it might make things clearer for the one reading this log message

Reply:
`write block` is fine because it's explicit about which method was called (see the below log)